streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignorTest.java

```diff
@@ -215,6 +215,55 @@ public void testAssignBasic() throws Exception {
     }
 
     @Test
+    public void testAssignBasicSingleGroupPartitionGrouper() throws Exception {
+        Properties props = configProps();
+        props.put(StreamsConfig.PARTITION_GROUPER_CLASS_CONFIG, SingleGroupPartitionGrouperStub.class);
+        StreamsConfig config = new StreamsConfig(props);
+        SingleGroupPartitionGrouperStub test = new SingleGroupPartitionGrouperStub();
+
+        TopologyBuilder builder = new TopologyBuilder();
+        builder.addSource("source1", "topic1");
+        builder.addProcessor("processor1", new MockProcessorSupplier(), "source1");
+        builder.addStateStore(new MockStateStoreSupplier("store1", false), "processor1");
+        builder.addSource("source2", "topic2");
+        builder.addProcessor("processor2", new MockProcessorSupplier(), "source2");
+        builder.addStateStore(new MockStateStoreSupplier("store2", false), "processor2");
+        List<String> topics = Utils.mkList("topic1", "topic2");
+
+        final Set<TaskId> prevTasks10 = Utils.mkSet(task0);
```

Comment:
I don't think we need these prevTasks and standbyTasks for this test. You can just pass `Collections.emptySet()` to the `Subscription`